scripts/release/controller/setup.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+if [ -z "${BUILDTIMESTAMP}" ]; then
+    date "+%Y%m%d_%H%M%S" > "${HOME}/buildtimestamp"
+    BUILDTIMESTAMP=$(cat "${HOME}/buildtimestamp")
+    export BUILDTIMESTAMP
+    echo run "${0}" with output to "${HOME}/buildlog_${BUILDTIMESTAMP}"
+    (bash "${0}" "${1}" "${2}" 2>&1) | tee "${HOME}/buildlog_${BUILDTIMESTAMP}"
+    exit 0
+fi
+
+date "+setup start %Y%m%d_%H%M%S"
+
+set -ex
+
+GIT_REPO_PATH=https://github.com/btoll/go-algorand
```

Comment:
You need to fix this.

Reply:
I'm still doing some builds, so this will be the last thing that I revert.